Sources/SwiftDoubleRatchet/SwiftDoubleRatchet.swift

```diff
@@ -1,22 +1,12 @@
 import Crypto
 import Foundation
 
-let infoMessage: Data = "MyRatchet".data(using: .utf8)!
+let infoMessage = "MyRatchet".data(using: .utf8)!
 
 public struct Ratchet<C: Curve> {
     struct PublicKeyNumber: Hashable {
-        static func == (lhs: PublicKeyNumber, rhs: PublicKeyNumber) -> Bool {
-            lhs.n == rhs.n && lhs.p == rhs.p
-        }
-        
-        func hash(into hasher: inout Hasher) {
-            p.hash(into: &hasher)
-            n.hash(into: &hasher)
-        }
-        
```

Comment:
This is exactly the same `Hashable` implementation that the compiler would synthesize.